cmd/dfs/cmd/server.go

```diff
@@ -75,6 +76,14 @@ can consume it.`,
 			_ = cmd.Help()
 			fmt.Println("\npostageBlockId is required to run server")
 			return
+		} else if len(postageBlockId) != 64 {
```

Comment:
why ?

Reply:
bee also validates batchId like this https://github.com/ethersphere/bee/blob/93b462981490004a405643bd4ca851f6817085b5/pkg/debugapi/postage.go#L404